docs/pages/development/create-development-builds.mdx

```diff
@@ -0,0 +1,175 @@
+---
+title: Create development builds
+---
+
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { Terminal } from '~/ui/components/Snippet';
+import { Tab, Tabs } from '~/ui/components/Tabs';
+import { BoxLink } from '~/ui/components/BoxLink';
+
+Development builds can be created with [EAS Build](/build/introduction/) or locally on your computer if you have Xcode and Android Studio. In the following sections, you will find information on how to create a development build and then install them on an emulator/simulator or a physical device to continue developing your app.
+
+## Prerequisites
+
+You need to initialize your project with Expo. If you still need to do so, see the [initializing steps](/get-started/create-a-new-app/#initializing-the-project) if you are starting a new project.
+
+## Install expo-dev-client
+
+To initialize a development build, you need to install the `expo-dev-client` library in your project:
+
+<Terminal cmd={['$ npx expo install expo-dev-client']} />
+
+## Build with EAS
+
+We recommend EAS Build to manage your native projects. It allows a smooth experience, especially if you do not have experience with Xcode and Android studio builds or have them installed locally on your computer.
+
+### Configure the project
+
+To get started, you need to configure your project for EAS Build by running the command:
+
+<Terminal cmd={['$ eas build:configure']} />
+
+> If you have not installed EAS CLI yet, we recommend to install it as global npm dependency by running `npm install -g eas-cli`.
+
+This command will create an [**eas.json**](https://docs.expo.dev/build/eas-json/) file. If you have an existing **eas.json** that does not have a `"development"` profile, modify it to add the profile as shown below:
+
+```json eas.json
```

Comment:
What do you think about consolidating the simulator config here

Reply:
I think it will be a good step in the user journey. I think most users would want to build for simulator or an APK before a device. I'll update this section.